base_classes/NXdata.nxdl.xml

```diff
@@ -219,29 +219,32 @@
 		
 		There are two older methods of associating 
 		each data dimension to its respective dimension scale.
-		Both are now out of date and
-		should not be used when writing new data files.
-		However, client software should expect to see data files
+		Neither should be used when writing new data files, but
+		client software should expect to see legacy data files
 		written with any of these methods.
 		
-		  * One method uses the ``axes`` 
-		    attribute to specify the names of each *dimension scale*.
+		  * One method adds the ``axes`` attribute to the data, rather than to
+		    the group. This is not recommended because the data field could be
+		    linked from another group, which does not contain all the 
+		    *dimension scales*.
 		
-		  * The oldest method uses the ``axis`` attribute on each
-		    *dimension scale* to identify
-		    with an integer the axis whose value is the number of the dimension.
+		  * The oldest method adds an ``axis`` attribute to each
+		    *dimension scale* set to an integer corresponding to the 
+		    corresponding dimension.
 	</doc>
-	<field name="VARIABLE" type="NX_NUMBER" nameType="any">
+	<field name="AXIS" type="NX_NUMBER" nameType="any">
 		<doc>
-	 		Dimension scale defining an axis of the data.
+	 		Dimension scale defining an axis of the data. There should be one
+	 		of these for each dimension of the ``DATA`` field.
 	 		Client is responsible for defining the dimensions of the data.
 	 		The name of this field may be changed to fit the circumstances.
 	 		Standard NeXus client tools will use the attributes to determine
 	 		how to use this field.
 	 	</doc>
 		<dimensions rank="1">
 			<doc>
-				A *dimension scale* must have a rank of 1 and has length ``n``.
+				A *dimension scale* must have a rank of 1 and has length ``n``
+				or ``n+1``.
```

Comment:
Okay, looks like that is a new addition. I am not sure about that.

Reply:
similar to the above comment, it depends on whether you are labelling the pixels or the edges.